packages/node-core/src/indexer/connectionPool.service.ts

```diff
@@ -49,6 +49,7 @@ export class ConnectionPoolService<T extends IApiConnectionSpecific<any, any, an
 
   async onApplicationShutdown(): Promise<void> {
     await Promise.all(Object.values(this.allApi).map((api) => api?.apiDisconnect()));
+    this.allApi = {};
```

Comment:
What's the reason for this change?

Reply:
Trying to reset all existing connection here, but I think need a better place